subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/SyncTaskIntegrationTest.groovy

```diff
@@ -394,84 +394,83 @@ class SyncTaskIntegrationTest extends AbstractIntegrationSpec {
     }
 
     @Issue("https://github.com/gradle/gradle/issues/9586")
-    def "change in case of input files and folders will still syncs properly"() {
+    def "change in case of input file will still sync properly"() {
```

Comment:
```suggestion
    def "change in case of input file will sync properly"() {
```

`still` implies we are comparing something to some earlier state, which is not the case here.